modules/ROOT/pages/runtime-patch-updates.adoc

```diff
@@ -0,0 +1,53 @@
+= Mule Runtime Patch Updates for Runtime Fabric 
+
+The following provides important information about Mule runtime engine patch updates for Runtime Fabric.
+
+== Monthly Date-Patch Updates 
+
+Once per month, MuleSoft releases a _date-patch update_, which includes backward-compatible bug fixes to address any issues discovered in Mule runtime engine.
+The monthly date-patch update might include security patches if the scheduled date is within the security SLA.
+
+The format of the version number for these updates is:
+
+_Major_._Minor_._Patch_ - _Date_ 
+
+Here's an example version number, including the patch date:
+
+`4.3.0:20210101`
+
+The monthly patches update applications only to the latest date-patch. They do not change the runtime version.
+
+== Runtime Fabric Runtime Patch Update Schedule 
+
+Runtime Fabric runs each application and gateway deployment on a separate Mule runtime instance. By managing the containerized Mule images used for these deployments, Runtime Fabric enables you to upgrade your Mule runtime version for each deployment on the following schedule:
+
+. On the first full week of the month, you have access to new date-patches of updated Mule runtimes.
+. During the next two weeks, you can redeploy your applications using the newer date-patch to validate its behavior. This is known as a grace period.
+. Beginning on the third full full week of the month, you must select the latest date-patch of your chosen Mule runtime version when deploying a new application, or when making a change to an existing application.
+
+[NOTE]
+During the grace period, you can select earlier date-patch versions until they are no longer available. If you select an earlier date-patch, and make any subsequent changes to that application, you may be required to update its date-patch version to a later release.
+
+== Update a Single App to the Latest Runtime Patch Release
+
+Before updating apps in the production environment, update them in the sandbox environment and run required tests. 
+
+To update an app to the latest date patch:
```

Comment:
Given the previous statement, is this and what follows AFTER the sandbox update and test?